functions.Rmd

```diff
@@ -14,6 +14,24 @@ row_adder()
 permutation()
 ```
 
+## Anonymous functions
+
+If using an anonymous function, prefer the new lambda syntax: `\(x) {}`.
+
+```R
+# Good
+map(xs, \(x) mean((x + 5)^2))
+
+# Bad
+map(xs, function(x) mean((x + 5)^2))
+map(xs, ~ mean((.x + 5)^2))
+```
+
+* Use informative argument names.
+* Only use an anonymous function if it fits on one line.
+* Avoid using `\()` except for anonymous functions.
+* Avoid using `\()` in a pipe.
```

Comment:
(aside, IINM `\()` was introduced specifically because of native pipes, so though I agree with the rule here, it is funny 😅)